Android/ConstraintLayout/build.cake

```diff
@@ -1,35 +1,65 @@
-#load "../../common.cake"
+var TARGET = Argument ("target", Argument ("t", "ci"));
 
-var TARGET = Argument ("target", Argument ("t", "Default"));
-
-var NUGET_VERSION = "2.0.0-alpha2";
-var AAR_VERSION = "2.0.0-alpha2";
+var NUGET_VERSION = "2.0.0-beta2";
+var AAR_VERSION = "2.0.0-beta2";
 
 var CONSTRAINT_LAYOUT_URL = string.Format ("https://dl.google.com/dl/android/maven2/com/android/support/constraint/constraint-layout/{0}/constraint-layout-{0}.aar", AAR_VERSION);
 var CONSTRAINT_LAYOUT_SOLVER_URL = string.Format ("https://dl.google.com/dl/android/maven2/com/android/support/constraint/constraint-layout-solver/{0}/constraint-layout-solver-{0}.jar", AAR_VERSION);
 
+Task("libs")
+	.IsDependentOn("externals")
+	.Does (() =>
+{
+	MSBuild("./source/ConstraintLayout.sln", c => {
+		c.Configuration = "Release";
+		c.Targets.Clear();
+		c.Targets.Add("Restore");
```

Comment:
Maybe rather use the `c.Restore = true;` and then don't worry about specifying the targets (ie: just use the default build)